railties/lib/rails/application/configuration.rb

```diff
@@ -128,6 +128,12 @@ def load_defaults(target_version)
             action_dispatch.use_cookies_with_metadata = true
           end
 
+          if respond_to?(:action_mailer)
+            ActiveSupport.on_load(:action_mailer) do
+              ActionMailer::Base.delivery_job = ::ActionMailer::MailDeliveryJob
```

Comment:
Don't we have a `config.action_mailer.delivery_job` scope?

Reply:
I thought `config.action_mailer.delivery_job =` didn't work for some reason in my sample app, but it appears to now. Probably `spring` playing tricks on me.